packages/cli/src/cmds/weakSubjectivityState.ts

```diff
@@ -0,0 +1,16 @@
+export const mainnetWeakSubjectivityState = {
+  stateRoot: "0xc87634afdb65d73221b9eff3d5383eee63a83289e91b6725494f0ff9c6c33dcc",
+  ipfsPath: "/ipfs/QmTbawGuYdfMD92XLViNPZrQdGcEmLmuETipG8YXxjnzqN/state.ssz",
+};
+
+// manually copied from the weakSubjectivityState.json ouput in beacon-state-upload
+export const praterWeakSubjectivityState = {
+  stateRoot: "0x9a4e6155e7341303e595529cd27529b4a61b7be6a38b7f437c68dc2dd9baf3da",
+  ipfsPath: "/ipfs/QmXM4yuVAZxd8jm2st6nfSy8gijnun4owePPDp3kqtHbyd/state.ssz",
+};
+
+// manually copied from the weakSubjectivityState.json ouput in beacon-state-upload
+export const pyrmontWeakSubjectivityState = {
+  stateRoot: "0xdab52683fc625aa156ca1861ddcabe642e56ffc3c0d6f04a11a9b721b92eb2d9",
+  ipfsPath: "/ipfs/QmPinARkXH7CnCq4A3fFwL6Gf46rubzbXZn1ggHdHULqRr/state.ssz",
+};
```

Comment:
How would we update this values every X time?

Reply:
Who is pinning those IPFS paths and with what replication factor?

Reply:
@3xtr4t3rr3str14l bump. This is the most critical question to answer around this PR

Reply:
Under this design, the pinning would be done by https://github.com/ChainSafe/beacon-state-upload/pull/3 every time a new weak subjectivity period is fetched via our API.  beacon-state-upload would be subscribed to the finalized checkpoints events stream via the API, and for every new finalized checkpoint, we would check to see if the WS API (via `lodestar/ws_epoch` custom API endpoint) gives us a new WS checkpoint.  If we have a new WS checkpoint, then we get the WS state from the API, and upload the WS checkpoint data and the WS state to IPFS & IPNS (which pins the data automatically via the IPFS HTTP API).  But under this setup, we would only use the IPFS link in this embedded file, for stability.

However, none of this may be relevant if we go with the newer solution in #2571 .  I'm open to suggestions on a fusion of the two strategies though, if that is desired.